src/components/footer/Footer.tsx

```diff
@@ -61,13 +61,21 @@ const Footer: React.FC<Props> = ({ hasContact }) => {
             <FooterNav />
           </Row>
 
-          <div className="pt-5 text-center">
-            <p>
+          <Row className="pt-5 justify-content-center">
+            <Col xs="auto">
+              <img src="/images/logos/kirdev.png" alt="Kir-Dev" height="46" />
+            </Col>
+            <Col xs="auto">
+              Website developed in <br />
+              collaboration with <a href="https://kir-dev.sch.bme.hu/">Kir-Dev</a>
```

Comment:
Erre is kéne egy translation, nem?

Reply:
Ja csinálok, csak a footer vmiért alapból is angol volt :D